synapse_data_warehouse/synapse_event/V2.49.0__create_synapse_event_schema.sql

```diff
@@ -0,0 +1,3 @@
+USE DATABASE {{ database_name }}; --noqa: JJ01,PRS,TMP
+
+CREATE OR REPLACE SCHEMA SYNAPSE_EVENT;
```

Comment:
Can we add a detailed description of what tables will exist in this schema?

Reply:
Sure, I can add it to either Confluence documentation or the Jira ticket. I don't think burying high-level outlines in the source code is very useful.

Reply:
No I mean you can use COMMENT and if should show up on snowflake UI